### DISCUSSION TITLE
ModuleNotFoundError: No module named 'model.bert'

### DISCUSSION THREAD

**Discussion by xxw11** [hpcaitech/ColossalAI#216] [Community | General]
![image](https://user-images.githubusercontent.com/55141908/153746115-d372e9f3-46d6-4506-95e3-1fe6db69b095.png)
跑example时出现了这个，是我哪里弄错了吗

**Comment by xxw11** [comment]
调整这个example在colosalai-main的位置
并改为
from model_zoo.gpt.gpt import GPTLMLoss

**Reply by FrankLeeeee** [reply]
Hi，如果你是pip install from source。应该是能直接import model_zoo的。如果不行，也可以mv model_zoo /path/to/xxx 然后export PYTHONPATH=/path/to/xxx，就可以import了

**Reply by xxw11** [reply]
测试了一下 按你说的方法确实有效 十分感谢
